conf/application.conf

```diff
@@ -0,0 +1,178 @@
+# Copyright 2020 HM Revenue & Customs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is the main configuration file for the application.
+# ~~~~~
+
+include "backend.conf"
+
+appName = mtd-api
+
+appUrl = "http://localhost:#port#"
```

Comment:
I believe the appName and appUrl can be changed to reflect the business api name and port